Tests/SonarScanner.MSBuild.PreProcessor.Test/PluginResourceCacheTests.cs

```diff
@@ -36,7 +36,7 @@ public class PluginResourceCacheTests
             ((Action)(() => new PluginResourceCache(basedir))).Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("basedir");
 
         [TestMethod]
-        public void Constructor_NullOrWhiteSpaceCacheDirectory_ThrowsArgumentNullException() =>
+        public void Constructor_NullOrWhiteSpaceCacheDirectory_ThrowsDirectoryNotFoundException() =>
```

Comment:
where is the null or whitespace cache directory? I see it's nonExistent

Reply:
I updated the name. Doing 100 things in parallel is killing my focus. Nice catch!